Cargo.toml

```diff
@@ -29,14 +29,9 @@ std = ["alloc"]
 # "active" portion of an `ArrayVec` or `SliceVec`.
 grab_spare_slice = []
 
-# features that require rustc 1.40
-# use Vec::append if possible in TinyVec::append - 1.37
-# DoubleEndedIterator::nth_back - 1.40
-rustc_1_40 = []
```

Comment:
This feature actually has to stay, even of it is on by default now. Downstream users will get an error if they had it on and we remove it. We can remove checking for it within the crate, we just need to leave it here as an empty "do nothing" feature for semver reasons.

Reply:
fixed